README.md

```diff
@@ -418,6 +418,14 @@ TestName | How to enable | Example
 
 *Performance tests use Gatling. You can find more information about the tool on their website https://gatling.io/.
 
+You can passthrough options to the frontend security scan script by using the URL_EXCLUSIONS variable in your Jenkinsfile. This allows you to customise zap proxy scans of your application.
+
+You can find an example [here](https://github.com/hmcts/idam-web-public/commit/aa5a40019959641ea1f355a9b6b66b56c99d34e0)
```

Comment:
I wouldn't make it a parameter, I would just hardcode it in the Jenkinsfile_nightly, simply the easiest to understand

Reply:
yeah